pyaedt/generic/design_types.py

```diff
@@ -49,6 +49,18 @@
 Simplorer = TwinBuilder
 
 
+def FilterSolutions(
+    version=None,
```

Comment:
Add type hints. The implementation_type especially will help developers locate the correct Enum type.